include/z64interface.h

```diff
@@ -116,27 +190,27 @@ typedef struct {
     /* 0x30A */ s16 perfectLettersColorTimer;
     /* 0x30C */ s16 perfectLettersTimer;
     struct {
-        /* 0x30E */ u8 unk_30E; // "h_gauge"
+        /* 0x30E */ u8 hGauge;
         /* 0x30F */ u8 bButton;
         /* 0x310 */ u8 aButton;
         /* 0x311 */ u8 tradeItems;
-        /* 0x312 */ u8 unk_312;
-        /* 0x313 */ u8 unk_313;
-        /* 0x314 */ u8 unk_314;
+        /* 0x312 */ u8 songOfTime;
+        /* 0x313 */ u8 SongOfDoubleTime;
```

Comment:
capital letter S? how could you you monster <img src="https://cdn.frankerfacez.com/emoticon/420214/2" height="24" alt="D:">

Reply:
Ah, because I went from `doubleSongOfTime` -> `SongOfDoubleTime`